libshpool/src/daemon/server.rs

```diff
@@ -931,6 +955,7 @@ impl Server {
                     .unwrap_or(DEFAULT_INITIAL_SHELL_PATH)),
             ),
             (s("SHPOOL_SESSION_NAME"), s(&header.name)),
+            (s("SHPOOL_SESSION_DIR"), session_dir.to_string_lossy().to_string()),
```

Comment:
env variable should really be OsStr/OsString for the sake of path correctness for different encodings (and avoid to_string_lossy). Probably not in scope of this PR though.

Reply:
Good point. I just did it now since I doubt I would remember to fix it in the future and it was fairly easy.